go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/project-copacetic/copacetic
 
-go 1.21
+go 1.22.0
 
```

Comment:
could we specify this as just 1.22?

Reply:
Sure, this is what the cli put, but can change it

Reply:
it should be ok to run go mod tidy, is it resulting in an error?

Reply:
It just puts the value back to 1.22.0 in go.mod